mobile-first.html

```diff
@@ -67,7 +67,7 @@ <h2 class="recent">My Recent Projects</h2>
   
             
           <div class="head-work">
```

Comment:
- Also just as the previous reviewer mentioned, the `See Project` button in this card is not properly styled as in the Figma design. Please ensure that the text fits properly in the button box and looks similar to the Figma Template. :100: 
- Your Design 
![image](https://user-images.githubusercontent.com/102699958/221249152-64d696db-168b-47a2-a901-84b563de94be.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/221249335-00140e53-df4d-46d8-aae9-1007d66ca29a.png)